notebooks/Notebook1.md

```diff
@@ -18,17 +28,76 @@ exciting milestones, preliminary results, etc. Feel free to include images
 code, and any other resources that you think will help clearly convey your
 design process.
 
+I added some very basic internal DSL implementations! A very barebones PhonemicString type, a very basic "=~" operator, and a very basic loop implementation. I can now build off of this. 
+
+Outside of coding, I also focused on the other aspects of getting started with the project : picking a host language, picking an internal/external DSL approach, talking experts in the Domain, etc.
+
+Picking a host language: 
+![image](https://user-images.githubusercontent.com/79538073/229378181-51c31208-7ec9-48f8-9b51-bccfe7d5f405.png)
+
+After a table analysis attached above, Scala came out the winner. 
+
+Internal DSL: I chose the internal DSL route.
+
+Domain Expert:
+I also spoke to my Linguistics professor about the DSL class, and I mentioned something about Regex which she was thoroughly confused by. So I thought about that. I think regex implementations can be really important while working with strings, but it is really unintuitive. Aside from the features I already intended on adding, I am also thinking about adding some regex features in the language that are more "readable", usable and better designed. 
+
+
 ## Questions
 
 **What is the most pressing issue for your project? What design decision do
 you need to make, what implementation issue are you trying to solve, or how
 are you evaluating your design and implementation?**
 
+Design decision: Currently my phonemic string looks like this: 
+
+"/text here/"
+
+And I think it would be more intuitive and concise if it looked like this instead: 
+
+/text here/
+
+The quotes are unnecessary and may confuse a Linguist (linguists use // anyways. Unlike programmers). It would also help differentiate the language from Scala, and differentiate the datatype from String. And make it more DSL-y by removing some flavor of the host language. 
```

Comment:
I like this decision especially if linguists are already used to using slashes anyway.